harness/determined/common/api/request.py

```diff
@@ -41,6 +40,15 @@ def maybe_upgrade_ws_scheme(master_address: str) -> str:
     else:
         return master_address
 
+def make_interactive_task_url(task_id: str, 
+        service_address: str, 
+        description:str, 
+        resource_pool: str,
+        task_type: str) -> str:
+    wait_path_url = service_address + "/notebooks/{}/events".format(task_id)
+    wait_page_url = "/det/wait/{}/{}?eventUrl={}&serviceAddr={}".format(task_type, task_id,wait_path_url,service_address)
+    notebook_web_url = "/det/interactive/{}/{}/{}/{}/{}".format(task_id,task_type,description,resource_pool,parse.quote_plus(wait_page_url))
```

Comment:
I am assuming we do not want to hard-code `/det/` here. If that is the case where should the actual value come from?

Reply:
From Hamid - the public web URL is an env var https://github.com/hamidzr/determined/blob/5272c09161e3e641d223d5fb1cd97ad4ff0f39c0/webui/react/Makefile#L2